manage-data/lifecycle/index-lifecycle-management/policy-view-status.md

```diff
@@ -0,0 +1,132 @@
+---
+navigation_title: View the lifecycle status of an index
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/update-lifecycle-policy.html
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# View the lifecycle status of an index [view-lifecycle-status]
```

Comment:
If I may request there is no built-in way to view this across all indices (only filter index management to a phase/errors but no aggregate stats) so I had requested https://github.com/elastic/elasticsearch/pull/99612/files which Dev didn't want to document before rather than improving the product but Support still ends up sending out that JQ as our only option for aggrgate "how's it doing"

Reply:
Since there are already issue opened for the health reports I'm going to mark this as resolved, since I don't think it affects the "Configure a lifecycle policy" page impacted by this PR.